Nodejs/Product/TestAdapter/TestFrameworks/Angular/karmaConfig.js

```diff
@@ -0,0 +1,42 @@
+﻿// @ts-check
+
+const path = require('path');
+
+const karmaConfigPath = path.resolve(process.env.PROJECTPATH, 'karma.conf.js');
```

Comment:
Will it be documented anywhere that this has to be in the project root?

Reply:
Based on new evidence this will change. Actually the karma config path is defined in `angular.json`. I'm working on implementing that.